src/Button.js

```diff
@@ -13,27 +13,23 @@ function fontSize({size = '14px', ...props}) {
   }
 }
 
-const ButtonBase = ({is: Tag, onClick, disabled, theme, ...rest}) => {
-  return <Tag disabled={disabled} onClick={disabled ? undefined : onClick} {...rest} />
```

Comment:
The `disabled` & `onClick` stuff here is unnecessary since HTML takes care of this natively

Reply:
Should we do anything special here to handle the case when the `as` prop is set to `"a"` and the button is disabled? I don't think the `disabled` attribute has any effect on links.

Reply:
Oh yeah good catch, I need to add that in